Makefile

```diff
@@ -16,22 +16,41 @@ TARGET = cc-runtime
 DESTDIR :=
 PREFIX := /usr/local
 BINDIR := $(PREFIX)/bin
-QEMUBINDIR := $(BINDIR)
 SYSCONFDIR := $(PREFIX)/etc
 LIBEXECDIR := $(PREFIX)/libexec
 LOCALSTATEDIR := $(PREFIX)/var
 SHAREDIR := $(PREFIX)/share
 
 CCDIR := clear-containers
+
 PKGDATADIR := $(SHAREDIR)/$(CCDIR)
+PKGLIBDIR := $(LOCALSTATEDIR)/lib/$(CCDIR)
+PKGRUNDIR := $(LOCALSTATEDIR)/run/$(CCDIR)
 PKGLIBEXECDIR := $(LIBEXECDIR)/$(CCDIR)
 
+KERNELPATH := $(PKGDATADIR)/vmlinux.container
+IMAGEPATH := $(PKGDATADIR)/clear-containers.img
+
+QEMUBINDIR := $(BINDIR)
+QEMUCMD := qemu-lite-system-x86_64
+QEMUPATH := $(QEMUBINDIR)/$(QEMUCMD)
+
+SHIMCMD := cc-shim
```

Comment:
`SHIMPATH` is being used by the build to set the default in the binary but also to set the shim path in the config file. It seems suboptimal to have both those files hard-code `/cc-shim` when the `Makefile` can define it once.

How about I simply remove  `SHIMPATH` from the `USER_VARS` list? This would effectively demote that variable to being considered "internal-only - don't touch!".